tests/src/rules/newline-after-import.js

```diff
@@ -852,14 +861,26 @@ ruleTester.run('newline-after-import', require('rules/newline-after-import'), {
       parserOptions: { ecmaVersion: 2015, considerComments: true, sourceType: 'module' },
     },
     {
-      code: `const foo = require('foo');\n\n\n// some random comment\nconst bar = function() {};`,
-      options: [{ count: 2, exactCount: true, considerComments: true }],
+      code: `var foo = require('foo-module');\nvar foo = require('foo-module');\n\n// Some random comment\nvar foo = 'bar';`,
+      output: `var foo = require('foo-module');\nvar foo = require('foo-module');\n\n\n// Some random comment\nvar foo = 'bar';`,
+      errors: [{
+        line: 2,
+        column: 1,
+        message: REQUIRE_ERROR_MESSAGE_MULTIPLE(2),
+      }],
+      parserOptions: { ecmaVersion: 2015, sourceType: 'module' },
+      options: [{ considerComments: true, count: 2 }],
+    },
+    {
+      code: `var foo = require('foo-module');\n\n/**\n * Test comment\n */\nvar foo = 'bar';`,
+      output: `var foo = require('foo-module');\n\n\n/**\n * Test comment\n */\nvar foo = 'bar';`,
       errors: [{
         line: 1,
         column: 1,
-        message: 'Expected 2 empty lines after require statement not followed by another require.',
+        message: REQUIRE_ERROR_MESSAGE_MULTIPLE(2),
       }],
-      parserOptions: { ecmaVersion: 2015 },
+      parserOptions: { ecmaVersion: 2015, sourceType: 'module' },
```

Comment:
why is sourceType module needed here? this is CJS, not ESM.
```suggestion
      parserOptions: { ecmaVersion: 2015 },
```

Reply:
this is my mistake. I will remove the module option.